V1 README update (deprecation notice) (#560)
microsoft/presidio: README.MD

---

## DEPRECATED: 
- This is the V1 version of Presidio. For the most recent version, go to [the `main` branch](https://github.com/microsoft/presidio/tree/main).

> - Note that the services in this version are not yet supported. 

Reviewer: ```suggestion
- Note that support in services in this version is ceased . 
```

Reviewer 2: / are no longer supported
